src/components/Engagements.js

```diff
@@ -17,7 +17,7 @@ export const mapStateToPropsEngagements = (state, ownProps) => {
     }
 }
 
-export const Engagements = ({dispatch, incidentId, engagements, user}) => {
+export const Engagements = ({ dispatch, incidentId, engagements, user }) => {
```

Comment:
I don't see why this revision needs to occur within this propType work. Is this related? If not, please restore to the master version.

Reply:
No problem.  Reverted